WooCommerce/Classes/ViewRelated/Orders/Cells/OrderTableViewCell.swift

```diff
@@ -91,6 +91,18 @@ final class OrderTableViewCell: UITableViewCell & SearchResultCell {
         super.prepareForReuse()
         paymentStatusLabel.layer.borderColor = UIColor.clear.cgColor
     }
+
+    override func updateConfiguration(using state: UICellConfigurationState) {
+        guard ServiceLocator.featureFlagService.isFeatureFlagEnabled(.splitViewInOrdersTab) else {
```

Comment:
We should call `super` here, since this is an override
```suggestion
        super.updateConfiguration(using: state)
        guard ServiceLocator.featureFlagService.isFeatureFlagEnabled(.splitViewInOrdersTab) else {
```

Reply:
Ah, thanks! Added here: https://github.com/woocommerce/woocommerce-ios/pull/11621/commits/186b35ef921e4fd1d199fd167d475a2134f9ca47